include/boost/http_proto/response.hpp

```diff
@@ -90,6 +93,12 @@ class BOOST_SYMBOL_VISIBLE
         http_proto::status sc,
         http_proto::version v);
 
+    /** Constructor
+    */
+    BOOST_HTTP_PROTO_DECL
+    response(
```

Comment:
should this be `explicit`?

Reply:
and can this be `noexcept`?

Reply:
Because the `http_proto::status` is an `enum class`, I don't think it's strictly required here.

I tried to making it file with something like `response res({200});` but I couldn't get it to compile in c++11 or c++20.

Reply:
can't hurt though, to communicate this to the reader